dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractJdbcTask.java

```diff
@@ -89,24 +90,33 @@ protected CSVFormat newCsvFormat(@Nonnull ResultSet rs) throws SQLException {
                 // Can we avoid nesting exceptions here?
                 throw new SQLException(
                         new MetadataDumperUsageException("Fatal Error. ResultSet does not have the expected column count: " + headerClass.getEnumConstants().length,
-                                Arrays.asList("If a custom query has been specified please confirm the selected columns match the following: ",
-                                        StringUtils.join(headerClass.getEnumConstants(), ", "))));
+                            Arrays.asList(
+                                "If a custom query has been specified please confirm the selected columns match the following: ",
+                                StringUtils.join(headerClass.getEnumConstants(), ", "))));
         } else {
             format = format.withHeader(rs);
         }
         return format;
     }
 
     @Nonnull
-    protected ResultSetExtractor<Void> newCsvResultSetExtractor(@Nonnull ByteSink sink, @CheckForSigned long count) {
+    protected ResultSetExtractor<Void> newCsvResultSetExtractor(@Nonnull ByteSink sink,
+        @CheckForSigned long count) {
```

Comment:
Optional: I would suggest to either keep the parameters in one line, or move each parameter to a separate line (both `sink` and `count`).